bundle/pom.xml

```diff
@@ -57,13 +57,16 @@
                             org.apache.sling.models.spi.injectorspecific.*;resolution:=optional, <!-- that package was added with Sling Models API 1.1 -->
                             com.day.cq.wcm.workflow.process;version="[6.0,8)", <!-- using a wider version range for forward compatibility -->
                             com.day.cq.mailer;version="[5.9,7)", <!-- using a wider version range for forward compatibility -->
+                            com.adobe.cq.sightly;version="[2.5,4)",
+                            com.day.cq.dam.api;version="[1.16,2)", <!-- using a wider version range for forward compatibility -->
```

Comment:
This is almost certainly the wrong import range. It should probably be `[1.16,1.18)` since any change to the `Asset` interface after `1.17` could break this code.